### PR TITLE
BaSP-TG-139: add token header to requests

### PR DESCRIPTION
**Pull request by fede230298**
Add Token to Headers in every request made to the server

### CODE REVIEW

**Review by sofihoj — APPROVED** [review]

**Review by ProDIGGY-vinilos — APPROVED** [review]

**Review by MaicolFreire001 — APPROVED** [review]

File: src/Components/Layout/AdminLayout/index.js [hunk 1]
```diff
@@ -49,9 +48,6 @@ const AdminLayout = () => {
             <Route exact path="/admin/tasks" component={Tasks} />
             <Route exact path="/admin/task-form" component={TasksForm} />
             <Route path="/admin/task-form/:id" component={TasksForm} />
-            <Route exact path="/admin/super-admins" component={SuperAdmins} />
-            <Route exact path="/admin/super-admin-form" component={SuperAdminForm} />
-            <Route path="/admin/super-admin-form/:id" component={SuperAdminForm} />
```
Comment (JuliMonte5) [inline src/Components/Layout/AdminLayout/index.js:54]: Delete the imports of this files (lines 13 and 14)

File: src/Components/Pages/Employee/ProjectList/index.js [hunk 2]
```diff
@@ -6,14 +6,15 @@ import MessageModal from 'Components/Shared/Modal/MessageModal';
 
 const ProjectsPage = () => {
   const userId = '636c1e8ddabe537336ae082a';
```
Comment (JuliMonte5) [inline src/Components/Pages/Employee/ProjectList/index.js:8]: Not the scope of this branch, but shouldn't we delete this var already?

File: src/Components/Layout/AdminLayout/index.js (hunk 1 above)
Comment (fede230298) [reply src/Components/Layout/AdminLayout/index.js:54]: Done

File: src/Components/Pages/Employee/ProjectList/index.js (hunk 2 above)
Comment (fede230298) [reply src/Components/Pages/Employee/ProjectList/index.js:8]: We still need it to display the employee search. For the time being at least

**Pull request merged**